_posts/en/2016-xx-xx-three-useful-hacks.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+
+title: Three useful hacks
+tip-number: 00
+tip-username: leandrosimoes 
+tip-username-profile: https://github.com/leandrosimoes
+tip-tldr: Three very useful and syntax sugar hacks to speed up your development.
+
+
+categories:
+    - en
+---
+
+#### Convert to boolean using "!!"
```

Comment:
this already exists http://www.jstips.co/en/converting-truthy-falsy-values-to-boolean/